pkg/api/node/formatter.go

```diff
@@ -61,6 +63,8 @@ func Formatter(request *types.APIRequest, resource *types.RawResource) {
 	resource.AddAction(request, listUnhealthyVM)
 	resource.AddAction(request, maintenancePossible)
 	resource.AddAction(request, powerActionPossible)
+	resource.AddAction(request, enableCPUManager)
+	resource.AddAction(request, disableCPUManager)
```

Comment:
can we please make this condition? please refer to usage below
we could only allow enableCPUManager when we know either last operation failed or cpu manager is disabled, and similarly allow disabling of manager if cpu manager is successfully enabled or failed?

Reply:
> we could only allow enableCPUManager when we know either last operation failed or cpu manager is disabled, and similarly allow disabling of manager if cpu manager is successfully enabled or failed?

Thanks for the comment, I'm not sure doing this is necessary, as we already have checks in validator to ensure the enable/disable cpu manager action is valid. If user triggers these actions in an unsupported scenario, the action will be no-op. So I'm not strongly feel we should make those action conditional, perhaps I missed something, is there something I should be aware of ?